cves/kernel/CVE-2013-3302.yml

```
@@ -105,22 +107,22 @@ vcc_instructions: |
 
   Place any notes you would like to make in the notes field.
 vccs:
-- commit: 6f49f46b187df34539f1e5df2469b8a541897700
-  note: Discovered automatically by archeogit.
-- commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
-  note: Discovered automatically by archeogit.
-- commit: b8eed28375a43e1c9aaa9d15af2a052aae0d0725
-  note: Discovered automatically by archeogit.
-- commit: 3e84469d0101456caceffc6b22218a49017fcd3f
-  note: Discovered automatically by archeogit.
+  - commit: 6f49f46b187df34539f1e5df2469b8a541897700
+    note: Discovered automatically by archeogit.
+  - commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
+    note: Discovered automatically by archeogit.
+  - commit: b8eed28375a43e1c9aaa9d15af2a052aae0d0725
+    note: Discovered automatically by archeogit.
+  - commit: 3e84469d0101456caceffc6b22218a49017fcd3f
+    note: Discovered automatically by archeogit.
```

If/once you have manually confirmed these:
```suggestion
  - commit: 6f49f46b187df34539f1e5df2469b8a541897700
    note: Manually Confirmed
  - commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
    note: Manually Confirmed
  - commit: b8eed28375a43e1c9aaa9d15af2a052aae0d0725
    note: Manually Confirmed
  - commit: 3e84469d0101456caceffc6b22218a49017fcd3f
    note: Manually Confirmed
```